tests/models/whisper/test_tokenization_whisper.py

```diff
@@ -339,6 +340,130 @@ def test_basic_normalizer(self):
         self.assertEqual(decoded_output_diacritics, expected_output_diacritics)
 
 
+    def test_decode_asr_with_word_level_timestamps(self):
+        # fmt: off
+        model_outputs = [
+            {
+                'stride': [30, 0, 5],
+                'tokens': torch.tensor([[
```

Comment:
Can we do everything in numpy? The Whisper tokenizer (and it's associated tests) should be hardware agnostic (PT, JAX, TF)